jit/v2/JIT_v2.0.0.yaml

```diff
@@ -95,18 +95,56 @@ paths:
       operationId: put-port-call
       summary: Initiates a new or updates a Port Call
       description: |
-        Description...
+        Creates or updates a **Port Call**. The caller must provide a unique portCallID (UUIDv4), which identifies the **Port Call**. The `portCallID` must remain consistent across all subsequent communications and linked **Terminal Calls**. If updating an existing **Port Call**, e.g. including the `portVisitReference` in case it was not provided earlier, the provided `portCallID` must match the existing record.
+
+        The **Port Call** consists of information about the location (which is required), ` business identifier for the Port (optional), static **Vessel** information (which also is required) and lastly if the provider is just sending the **Port Call** as information (`isFYI= true`).
```

Comment:
The **Port Call includes**:

- Location information (required): `UNLocationCode`.
- Static vessel information (required): `vessel`.
- An optional business identifier for the port visit: `portVisitReference`.
- The ability to send the record with an informational purpose only, using `isFYI`=`true`.